src/Table/Table.js

```diff
@@ -200,7 +200,7 @@ class Table extends React.Component {
           </td>
         );
 
-        if (column.dontCache) {
+        if (!column.cacheCell) {
```

Comment:
If someone doesn't define this prop, will it be a problem as this will conditional will be truthy?